CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [2.0.0](https://www.github.com/googleapis/java-spanner/compare/v1.61.0...v2.0.0) (2020-09-16)
+
+
+### ⚠ BREAKING CHANGES
+
+* ensure next release is major (#428)
+
+### Features
+
+* add lazy initializer ([#423](https://www.github.com/googleapis/java-spanner/issues/423)) ([e8522b9](https://www.github.com/googleapis/java-spanner/commit/e8522b9955c4a19fa7d6297fd463e9d2521dff92))
+
+
+### Bug Fixes
+
+* fix aborted handling of batchUpdateAsync ([#421](https://www.github.com/googleapis/java-spanner/issues/421)) ([6154008](https://www.github.com/googleapis/java-spanner/commit/61540085c971d7885e4938b486e051a1ed9cf35f))
+* Remove Guava ImmutableList from API surface ([#411](https://www.github.com/googleapis/java-spanner/issues/411)) ([b35304e](https://www.github.com/googleapis/java-spanner/commit/b35304ede5c980c3c042b89247058cc5a4ab1488))
```

Comment:
Can we move this to the breaking changes section? We have to do this manually since the bot can't figure it out based on the PR title.

Reply:
Weird, I had done that before and my changes got overwritten. Moved it again, thanks for pointing that out.